.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/dask-cuda/labels
+
+Python:
+ - python/**
+ - notebooks/**
+
+libdaskcuda:
+ - cpp/**
+
+CMake:
+ - **/CMakeLists.txt
+ - **/cmake/**
+
+Java:
+ - java/**
```

Comment:
Do we need these for this library? Asking as it seems unlikely we would add C++ or Java (and for that matter CMake) since the main goal is to build on top of Dask in Python.